packages/arb-token-bridge-ui/src/components/App/App.tsx

```diff
@@ -137,7 +140,10 @@ const AppContent = (): JSX.Element => {
       <HeaderOverrides {...headerOverridesProps} />
 
       <HeaderContent>
-        <HeaderNetworkInformation />
+        <NetworkSelectionContainer>
+          <HeaderNetworkInformation />
+        </NetworkSelectionContainer>
```

Comment:
![image](https://user-images.githubusercontent.com/13184582/196683900-f8a6c75a-e04b-46ed-bf94-0c641b090dca.png)
minor UI regression using `NetworkSelectionContainer`
it seems to only happen with Nova, probably because of the logo's height